src/dispatch/case/flows.py

```diff
@@ -176,9 +176,18 @@ def create_conversation(case: Case, conversation_target: str, db_session: Sessio
 
 def update_conversation(case: Case, db_session: SessionLocal):
```

Comment:
```suggestion
def update_conversation(case: Case, db_session: Session):
```

Maybe not in this PR but we should replace all of the `SessionLocal` with `Session` since `SessionLocal` just `Any`

Reply:
Ack.